src/main/java/dev/slimevr/vr/MountingCalibration.java

```diff
@@ -0,0 +1,154 @@
+package dev.slimevr.vr;
+import io.eiren.util.collections.FastList;
+import io.eiren.util.logging.LogManager;
+import dev.slimevr.VRServer;
+import dev.slimevr.vr.trackers.IMUTracker;
+import dev.slimevr.vr.trackers.ReferenceAdjustedTracker;
+import dev.slimevr.vr.trackers.Tracker;
+import dev.slimevr.vr.trackers.TrackerConfig;
+
+import com.jme3.math.FastMath;
+import com.jme3.math.Quaternion;
+
+public class MountingCalibration {
+
+	private final VRServer server;
+	private FastList<Quaternion> trackerOrientationIdle = new FastList<>();
+	private FastList<Tracker> allTrackers = new FastList<>();
+	private int trackerNumber;
+	private static float TENTH_THIRD = 0.03333f, FIFTH_THIRD = 0.06666f;
```

Comment:
Please don't use magic constants like this. If you need 3%, use something like 3f / 100f and 6f / 100f instead.